pom.xml

```diff
@@ -515,7 +515,7 @@
     <jackson-guava-version>2.4.2</jackson-guava-version>
     <logback-version>1.0.1</logback-version>
 
-    <junit-version>4.8.1</junit-version>
+    <junit-version>4.8.2</junit-version>
```

Comment:
Need this for `org.junit.Assert.assertThat(...)` and `org.hamcrest.CoreMatchers.instanceOf(Class<?>)`